pom.xml

```diff
@@ -39,6 +39,8 @@
     <module>hawkular-wildfly-monitor</module>
     <module>hawkular-wildfly-monitor-feature-pack</module>
     <module>hawkular-agent-itest-parent</module>
+    <module>extension-installer</module>
```

Comment:
rename this module something more descriptive like "wildfly-module-installer" or "wildlfy-subsystem-extension-installer" ... "extension-installer" has no indication that its for wildfly